src/Compilers/CSharp/Portable/Symbols/PublicModel/PropertySymbol.cs

```diff
@@ -58,6 +58,11 @@ IMethodSymbol IPropertySymbol.SetMethod
             get { return _underlying.SetMethod.GetPublicSymbol(); }
         }
 
+        IFieldSymbol IPropertySymbol.BackingField
+        {
+            get { return (_underlying as SourcePropertySymbolBase)?.BackingField.GetPublicSymbol(); }
```

Comment:
Would it be desirable to pull `SourcePropertySymbolBase.BackingField` up to `PropertySymbol` as either abstract or virtual returning null?